dev/io.openliberty.microprofile.faulttolerance.4.1.internal_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -11,12 +11,12 @@
 
     <groupId>io.openliberty.microprofile.faulttolerance</groupId>
     <artifactId>tck.runner.tck</artifactId>
-    <version>4.1</version>
+    <version>4.1.1</version>
```

Comment:
This technically can stay 4.1.  If you look at the 4.0 version of this FAT bucket, it was left to 4.0 for this one.

Reply:
Ok, fixed.  Thanks for letting me know!